config.yml

```diff
@@ -338,6 +338,52 @@ nodes:
 
           [1, 2, 3]
           ^^^^^^^^^
+  - name: HashNode
+    child_nodes:
+      - name: opening
+        type: token
+      - name: elements
+        type: node[]
+      - name: closing
+        type: token
+    location: opening->closing
+    comment: |
+      Represents an hash literal
+
+          { a => b }
+          ^^^^^^^^^^
+  - name: AssocNode
+    child_nodes:
+      - name: opening
+        type: token
+      - name: key
+        type: node
+      - name: operator
+        type: token
+      - name: value
+        type: node
+      - name: closing
+        type: token
+    location: opening->closing
+    comment: |
+      Represents an hash key/value combo
+
+          { a => b }
+            ^^^^^^
+  - name: AssocSplat
+    child_nodes:
+      - name: opening
```

Comment:
Same here, I think we should call this `operator` and then you can drop the `closing` token, for location it can be `operator->value`.